x-pack/plugins/ml/public/util/custom_url_utils.js

```diff
@@ -67,7 +67,7 @@ export function getUrlForRecord(urlConfig, record) {
 // fullUrl is the complete URL, including the base path, with any dollar delimited tokens
 // from the urlConfig having been substituted with values from an anomaly record.
 export function openCustomUrlWindow(fullUrl, urlConfig) {
-  if (isKibanaUrl(urlConfig) === true) {
+  if (/^(?:[a-z]+:)?\/\//i.test(urlConfig.url_value) === false) {
```

Comment:
would it be worth wrapping this regex check in an `isExternalUrl` function?
Or maybe just adding a comment saying that it's looking to see if the url starts with a protocol scheme.

Reply:
I've added some extra comments around the use of the regex.